app/src/main/res/layout/comment_header.xml

```diff
@@ -0,0 +1,112 @@
+        app:layout_constraintTop_toTopOf="parent"
+        tools:src="@tools:sample/backgrounds/scenic" />
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:background="?colorPrimary"
+        android:padding="14dp"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@id/commentHeaderImage">
+
+        <TextView
+            android:id="@+id/commentHeaderTitle"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:textColor="@color/md_white_1000"
+            android:textSize="18sp"
+            tools:text="Pretty Corn Field" />
+
+        <TextView
+            android:id="@+id/commentHeaderAuthorBy"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:text="by"
+            android:textColor="@color/md_white_1000"
+            app:layout_constraintBottom_toBottomOf="@id/commentHeaderAuthor"
+            app:layout_constraintTop_toTopOf="@id/commentHeaderAuthor" />
+
+        <TextView
+            android:id="@+id/commentHeaderAuthor"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:paddingStart="4dp"
+            android:paddingTop="2dp"
+            android:paddingEnd="4dp"
+            android:paddingBottom="3dp"
+            android:textColor="@color/md_white_1000"
+            android:textStyle="bold"
+            app:layout_constraintStart_toEndOf="@id/commentHeaderAuthorBy"
+            app:layout_constraintTop_toBottomOf="@id/commentHeaderTitle"
+            tools:text="momodao" />
+
+        <TextView
+            android:id="@+id/commentHeaderDateTime"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:textColor="@color/md_white_1000"
+            app:layout_constraintBottom_toBottomOf="@id/commentHeaderAuthor"
+            app:layout_constraintStart_toEndOf="@id/commentHeaderAuthor"
+            app:layout_constraintTop_toTopOf="@id/commentHeaderAuthor"
+            tools:text=" 19 hours ago" />
+
+        <TextView
+            android:id="@+id/commentPoint"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:alpha="0.7"
+            android:textColor="@color/md_white_1000"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@id/commentHeaderAuthor"
+            tools:text="123 " />
+
+        <TextView
+            android:id="@+id/commentPointLabel"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:alpha="0.7"
+            android:text=" points,"
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/worker8/Pixels/blob/086aebf09732a206da871abbc219e4c95753b456/app/src/main/res/layout/comment_header.xml#L86"></span>Hardcoded string “ points,”, should use <code>@string</code> resource</td>
    </tr>
  </tbody>
</table>